packages/server/src/arpa_reporter/db/uploads.js

```diff
@@ -97,6 +97,12 @@ async function createUpload (upload, trns = knex) {
   return inserted
 }
 
+async function cacheRecords(upload, records, trns = knex) {
+  return trns('uploads')
+    .where('id', upload.id)
+    .update({ parsed_data: JSON.stringify(records) })
```

Comment:
I wonder actually if it makes sense to add a `cached_date` and/or `cached_version` property here to anticipate needing to do more advanced invalidation or update/change the json structure in the future (to Andrew's point in Slack?)

I'm not sure of best practices here but I've seen some combination of date storage and light cache versioning be potentially helpful for similar scenarios

Reply:
yeah, a cached date here would be super useful. Especially so that if we have a breaking change in how data is parsed, we should be able to query for any cached versions from before that deploy and refresh them (or at least consider them invalid so they'll refresh manually later). I'll add that column!